.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": "plugin:react/recommended",
```

Comment:
Is this linter also relevant to the backend module or only for the frontend module?

Reply:
The current state of the config file only adds React and Typescript as linters, so it's mostly handling the front end part.

My guess is there should be additional entries in the config file for back-end which highly depends on the tech stack used in the backend system.